sparrow_datums/types.py

```diff
@@ -35,6 +35,21 @@ class PType(enum.Enum):
     (x, y, w, h) in relative pixel coordinates
     """
 
+    """
+    Keypoint in relative space
+    """
+    relative_kp = "relative_kp"
```

Comment:
Let's call these `relative_xy` and `absolute_xy` just to match the box type pattern

Reply:
will do